### DISCUSSION TITLE
Override tflint arguments

### DISCUSSION THREAD

**Discussion by neilhwatson** [gruntwork-io/knowledge-base#751] [How do I]
A tflint before-hook appears to ignore additional parameters. I want to be able to set tflint's `--minimum-failure-severity`. Terragrunt does not allow this and tflint does not have config file option for this. What is the solution? Anything better than a wrapper?

---

<ins datetime="2023-07-05T17:25:38Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110324">Tracked in ticket #110324</a></p>
</ins>

**Comment by denis256** [comment]
Hi,
this looks like a bug - will check if can be fixed somehow
but so far I don't see something better than to have a script named "tflint" which inside will call tflint executable with additional parameters

**Comment by denis256** [comment]
Hello,
added possibility pass additional tflint arguments in https://github.com/gruntwork-io/terragrunt/releases/tag/v0.49.0
